src/api/controllers/entry/search.js

```diff
@@ -33,7 +33,7 @@ const Search = (userModel, teamModel, tournamentModel) => async (request, respon
       },
     });
 
-    if (users.length > 1) {
+    if (users.length !== 1) {

```

Comment:
Not found ça correspond à une liste d'utilisateur vide, donc `user.length === 0` non ?